src/views/viewer/SearchView/SearchResults/SearchResults.tsx

```diff
@@ -99,7 +99,9 @@ export const SearchResults: React.FC<SearchResultsProps> = React.memo(({ query }
       </SearchControls>
       <Results filtersOpen={isFiltersOpen}>
         <LimitedWidthContainer big>
-          {!loading && channels.length === 0 && videos.length === 0 && !!query ? (
+          {!loading &&
+          ((videos.length === 0 && selectedIndex === 0) || (channels.length === 0 && selectedIndex === 1)) &&
+          !!query ? (
```

Comment:
What's is selectedIndex in this context? Is it a selected tab index?
If yes, could we change this to `selectedTabIndex`?
Also, could we store this long expression in a variable?  I think it should be more readable.